include/mtl_api.h

```diff
@@ -503,6 +503,10 @@ struct mtl_init_params {
    * Suggest using rss (L3 or L4) for rx packets direction.
    */
   enum mt_rss_mode rss_mode;
+  /**
+   * Force to use DPDK IOVA PA mode.
+   */
+  bool iova_pa;
```

Comment:
Can we use a "enum mt_iova_mode" here? Currently it only has two options, VA and PA. In future it may possible has third option, like SIOV mode.

Reply:
Also the args should has one argument, like "--iova_mode pa" if we take this enum change.

Reply:
> Can we use a "enum mt_iova_mode" here? Currently it only has two options, VA and PA. In future it may possible has third option, like SIOV mode.

can we reuse rte_iova_mode?

Reply:
No, rte_iova_mode is defined in DPDK header file. We can't export it in our header.